presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcWarnings.java

```diff
@@ -225,6 +215,30 @@ public void testSqlWarning()
         assertWarningsEqual(warning.getNextWarning().getNextWarning(), toPrestoSqlWarning(warnings.get(2)));
     }
 
+    private static Set<WarningEntry> getStatementWarnings(Statement statement, Future<?> future)
+            throws Exception
+    {
+        Set<WarningEntry> warnings = new HashSet<>();
+        SQLWarning warning = null;
+        while (!future.isDone()) {
+            SQLWarning nextWarning;
```

Comment:
Technically you don't  need two variables: `warning` and `nextWarning`. One would do.